src/footer/_footer.scss

```diff
@@ -0,0 +1,192 @@
+@include govuk-exports("footer") {
+
+  $govuk-footer-background: $govuk-grey-3;
+  $govuk-footer-border-top: #a1acb2;
+  $govuk-footer-border: $govuk-grey-2;
+  $govuk-footer-text: #454a4c;
+  $govuk-footer-link: $govuk-footer-text;
+  $govuk-footer-link-hover: #171819;
+
+  // Based on the `govuk-file-url("govuk-crest.png");` image dimensions.
+  $govuk-footer-crest-image-width-2x: 250px;
+  $govuk-footer-crest-image-height-2x: 204px;
+  // Half the 2x image so that it fits the regular 1x size.
+  $govuk-footer-crest-image-width: ($govuk-footer-crest-image-width-2x / 2);
+  $govuk-footer-crest-image-height: ($govuk-footer-crest-image-height-2x / 2);
+
+  .govuk-c-footer {
+    @include govuk-font-regular;
+    @include govuk-typography-responsive($govuk-font-16);
+
+    padding-top: $govuk-spacing-scale-7;
+    padding-bottom: $govuk-spacing-scale-5;
+
+    border-top: 1px solid $govuk-footer-border-top;
+    color: $govuk-footer-text;
+    background: $govuk-footer-background;
+  }
+
+  .govuk-c-footer__link {
+    @include govuk-focusable-fill;
+
+    &:link,
+    &:visited {
+      color: $govuk-footer-link;
+    }
+
+    &:hover,
+    &:active {
+      color: $govuk-footer-link-hover;
+    }
+  }
+
+  .govuk-c-footer__section-break {
+    margin: 0; // Reset `<hr>` default margins
+    @include govuk-responsive-margin($govuk-spacing-responsive-8, "bottom");
+    border: 0; // Reset `<hr>` default borders
+    border-bottom: 1px solid $govuk-footer-border;
+  }
+
+  .govuk-c-footer__meta::after {
+    content: " ";
+    display: table;
+    clear: both;
+  }
+
+  .govuk-c-footer__meta {
+    display: flex; // Support: Flexbox
+    margin-right: -($govuk-gutter / 2);
```

Comment:
should we move away from old gutter and use $govuk-spacing-scale-6 directly?

Reply:
The gutter has a direct relationship between the margins that make up the grid system and containers so it felt proper to use it here.

I'll defer to @dashouse for this

Reply:
I think it's probably right to link it to the grid layout as we'd want them to change together if they changed...but we do have a variable for `$govuk-gutter-half` already so you can use that.